examples/simple_eth.c

```diff
@@ -0,0 +1,75 @@
+#include <ingenialink/ingenialink.h>
+#include <ingenialink/registers.h>
+#include <time.h>
+#include <stdio.h>
+
+
+int main(int argc, const char *argv[])
+{
+	il_net_t *net = NULL;
+	il_servo_t *servo = NULL;
+	char* dict_path = "XXX";
+	char* target_ip = "192.168.2.22";
+	int port = 1061;
+	int protocol = 2;
+	int r;
+	r = il_servo_lucky_eth(IL_NET_PROT_ETH, &net, &servo, dict_path, target_ip, port, protocol);
+
+	if (r < 0) {
+		fprintf(stderr, "CONNECTION FAIL %d\n", r);
+		return -1;
+	}
+
+	int32_t actual_position_buf;
+    const il_reg_t IL_REG_POS_ACT = {
+		.subnode = 1,
+        .address = 0x0030,
+        .dtype = IL_REG_DTYPE_S32,
+        .access = IL_REG_ACCESS_RO,
+        .phy = IL_REG_PHY_NONE,
+        .range = {
+            .min.s32 = INT32_MIN,
+            .max.s32 = INT32_MAX
+        }
+    };
+
+
+	float bus_voltage_value_buf;
+    const il_reg_t IL_REG_BUS_VOLT = {
+		.subnode = 1,
+        .address = 0x0060,
+        .dtype = IL_REG_DTYPE_FLOAT,
+        .access = IL_REG_ACCESS_RO,
+        .phy = IL_REG_PHY_NONE,
+        .range = {
+            .min.s32 = -100000000,
+            .max.s32 = 100000000
+        }
+    };
+
+	int n_iter = 1000;
+	time_t init_t;
+	init_t = time(0);
+
+	for (int i = 0; i<n_iter; i++) {
```

Comment:
Follow [style guide ](https://github.com/ingeniamc/ingenia-style-guide/blob/master/c_cpp_style.md#types)for loops.